apigee/adapter.go

```diff
@@ -391,27 +391,24 @@ func (h *handler) HandleQuota(ctx context.Context, inst *quotaT.Instance, args a
 func resolveClaims(log adapter.Logger, claimsIn map[string]string) map[string]interface{} {
 	var claims = map[string]interface{}{}
 	for _, k := range auth.AllValidClaims {
-		if v, ok := claims[k]; ok {
+		if v, ok := claimsIn[k]; ok {
```

Comment:
We apparently don't have test evidence that this was broken (or now works)?

Reply:
Yes the test was broken too. When I fixed the test, this started failing, so I fixed this too.

Reply:
d'oh! I see it now. Thanks.